src/metabase/driver/athena.clj

```diff
@@ -69,7 +78,7 @@
     :map        :type/*
     :smallint   :type/Integer
     :string     :type/Text
-    :struct     :type/*
+    :struct     :type/Dictionary
     :timestamp  :type/DateTime
     :tinyint    :type/Integer
     :varchar    :type/Text} database-type))
```

Comment:
Can/should we pull in the `column->base-type` definitions from `schema_parser`?

I'm not very familiar with Clojure so unsure if this is a valid suggestion.

Reply:
Yes, i did it on this commit [https://github.com/dacort/metabase-athena-driver/pull/10/commits/e1fcba3cac8eb3cc93f799f68332bb6ca09d812e](url)